.github/integration-in-memory-tests.yml

```
@@ -504,6 +504,8 @@ runner-test-matrix:
     triggers:
       - PR Integration CCIP Tests
       - Nightly Integration CCIP Tests
-    test_cmd: cd smoke/ccip && go test ccip_ton_messaging_test.go -timeout 20m -test.parallel=1 -count=1 -json
+    test_cmd:
+      cd .. &&  make install-plugins && cd integration-tests/smoke/ccip && go test ccip_ton_messaging_test.go -timeout 20m -test.parallel=1 -count=1 -json
```

`cd .. &&  make install-plugins &&`

This seems redundant, Aptos tests also use the plugin and don't need this? You probably had to do it because `install_path` was wrong
```suggestion
      cd smoke/ccip && go test ccip_ton_messaging_test.go -timeout 20m -test.parallel=1 -count=1 -json
```